samples-generator/bin/3d-tiles-samples-generator.js

```diff
@@ -7,6 +7,9 @@ var gltfPipeline = require('gltf-pipeline');
 var path = require('path');
```

Comment:
Point clouds with per-point properties should have a feature id attribute and implicit accessor. Notice that `_FEATURE_ID_0` is missing from the top-level attributes below. See `_FEATURE_ID_0` in [Point cloud with per-point properties and per-group properties](https://github.com/CesiumGS/glTF/tree/batch-table/extensions/2.0/Vendor/CESIUM_3dtiles_feature_metadata#point-cloud-with-per-point-properties-and-per-group-properties).

        {
          "attributes": {
            "POSITION": 0,
            "COLOR_0": 1
          },
          "mode": 0,
          "extensions": {
            "CESIUM_3dtiles_feature_metadata": {
              "attributes": {
                "_FEATURE_ID_0": 0
              }
            }
          }
        }